dataset_builders/pie/argmicro/README.md

```diff
@@ -0,0 +1,31 @@
+# PIE Dataset Card for "ArgMicro"
+
+This is a [PyTorch-IE](https://github.com/ChristophAlt/pytorch-ie) wrapper for the
+[ArgMicro Huggingface dataset loading script](https://huggingface.co/datasets/DFKI-SLT/argmicro).
+
+## Data Schema
+
+The document type for this dataset is `ArgMicroDocument` which defines the following data fields:
+
+- `id` (str)
+- `text` (str)
+- `topic_id` (str, optional)
+- `stance` (str, optional)
+- `metadata` (dictionary, optional)
+
+and the following annotation layers:
+
+- `edus` (annotation type: `Span`, target: `text`)
+- `adus` (annotation type: `LabeledAnnotationCollection`, target: `edus`)
+- `relations` (annotation type: `MultiRelation`, target: `adus`)
+
+See [here](https://github.com/ChristophAlt/pytorch-ie/blob/main/src/pytorch_ie/annotations.py) for the annotation type definitions.
+
+## Document Converters
+
+The dataset provides document converters for the following target document types:
+
+- `pytorch_ie.documents.TextDocumentWithLabeledSpansAndBinaryRelations`
```

Comment:
this is also not trivial how this works, can you add some subitems that shortly explains how we get the spans and binary relations from the given annotations?

Reply:
edit in 8e97ed3

Reply:
As above, see https://github.com/ArneBinder/pie-datasets/pull/56/commits/3723f60d2e66a35374fedda810fb10eb5528255b. Please look into that.

Reply:
made further changes in 2a9c40a, please have a look